src/modules/UI.js

```diff
@@ -0,0 +1,81 @@
+import Modal from './ModalInterface.js';
+
+export default class UI {
+  static async #privateGetShows() {
+    const response = await fetch('https://api.tvmaze.com/shows');
+    const data = await response.json();
+    return data;
+  }
+
+  static async getPopularShows(weight) {
+    const allShows = await UI.#privateGetShows();
+    const popularShows = await allShows.filter((show) => show.weight > weight);
+    return popularShows;
+  }
+
+  static imageListener() {
+    const images = document.querySelectorAll('.show-image');
+    if (images) {
+      images.forEach((image) => {
+        image.addEventListener('click', () => {
+          Modal.displayModal(image.dataset.showId);
+        });
+      });
+    }
+  }
+
+  static commentsListener() {
+    const comments = document.querySelectorAll('.comments-btn');
+    if (comments) {
```

Comment:
**Changes Required**
- [ ] Thanks for implementing these functionality. However, I suggest we should prefer the use of  `ES6 Arrow` functions. Let us try and refactor these functions using the `ES6 Arrow Functions.

Reply:
This is a request about your personal preference? I'm using ES6 class syntax for about 4 weeks and you are the first TSE that asks me to change this, could you explain me better why arrox functions are required here?

Reply:
For the future reviewer, if my previous question is not answered I'll not do this suggestion.